### PR TITLE
Env setup

### PR DESCRIPTION
**Pull request by priyanka-TL**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+TELEMETRY_DATASOURCE_NAME = "TELEMETRY_DATASOURCE_NAME"
+SURVEY_DATASOURCE_NAME = "SURVEY_DATASOURCE_NAME"
+SURVEY_EVIDENCE_DATASOURCE_NAME = "SURVEY_EVIDENCE_DATASOURCE_NAME"
+CONTENT_REPORT_THRESHOLD = 5
+ENTITY_SCORE_REPORT_THRESHOLD = 5
+OBSERVATION_SCORE_REPORT_THRESHOLD = 2
+ASSESSMENT_SUBMISSION_REPORT_THRESHOLD = 3
+EVIDENCE_THRESHOLD = 3
+
+# cassandra
```
Comment (aks30) [inline .env.sample]: @deepakr123 - Remove this and related code.

File: .env.sample [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+sunbird_keycloak_auth_server_url="https://dev.shikshalokam.org/auth"
+sunbird_keycloak_realm="sunbird"
+sunbird_keycloak_client_id="portal"
+sunbird_keycloak_public=true
+sunbird_cache_store="memory"
+sunbird_cache_ttl=1800
+
+URL_PREFIX = "api/v1"
+
+#KENDRA service
```
Comment (aks30) [inline .env.sample]: @deepakr123 - Please correct the comment

File: app.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
+require("dotenv").config();
```
Comment (aks30) [inline app.js:1]: @deepakr123 - Check and implement envVariables.js

Comment (deepakr123) [reply app.js:1]: resolved

**Pull request merged**